doc/ikfast/ikfast_tutorial.rst

```diff
@@ -73,210 +52,83 @@ Inside your catkin workspace's ``./src`` directory: ::
-
-From experience we recommend 5 decimal places, but if the OpenRAVE IKFast generator takes to long to find a solution, lowering the number of decimal places should help.
-
-To see the links in your newly generated Collada file
-
-You may need to install package **libsoqt4-dev** to have the display working: ::
-
- openrave-robot.py "$MYROBOT_NAME".dae --info links
-
-This is useful if you have a 7-dof arm and you need to fill in a ``--freeindex`` parameter, discussed later.
-
-To test your newly generated Collada file in OpenRAVE: ::
-
- openrave "$MYROBOT_NAME".dae
-
-You should see your robot.
-
-.. image:: openrave_panda.png
-   :width: 700px
-
-Create IKFast Solution CPP File
--------------------------------
-Once you have a numerically rounded Collada file its time to generate the C++ .h header file that contains the analytical IK solution for your robot.
+  rosrun xacro xacro -o $MYROBOT_NAME.urdf $MYROBOT_NAME.urdf.xacro
 
 Select IK Type
 ^^^^^^^^^^^^^^
-You need to choose which sort of IK you want. See `this page <http://openrave.org/docs/latest_stable/openravepy/ikfast/#ik-types>`_ for more info.  The most common IK type is *transform6d*.
+You need to choose which type of IK you want to solve for. See `this page <http://openrave.org/docs/latest_stable/openravepy/ikfast/#ik-types>`_ for more info.
+The most common IK type is *transform6d*.
 
 Choose Planning Group
 ^^^^^^^^^^^^^^^^^^^^^
-If your robot has more than one arm or "planning group" that you want to generate an IKFast solution for, choose one to generate first. The following instructions will assume you have chosen one ``<planning_group_name>`` that you will create a plugin for. Once you have verified that the plugin works, repeat the following instructions for any other planning groups you have. For example, you might have 2 planning groups: ::
+If your robot has more than one arm or "planning group" that you want to generate an IKFast solver for, you need to repeat the following process for each group.
+The following instructions will assume you have chosen one ``<planning_group_name>``. Furthermore, you need to know the names of the base link and the endeffector link of the chain to solve for.
 
- <planning_group_name> = "left_arm"
- <planning_group_name> = "right_arm"
+Generate IKFast MoveIt plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To make it easy to use copy/paste for the rest of this tutorial. Set a ``PLANNING_GROUP`` environment variable. eg: ::
+To generate the IKFast MoveIt plugin, issue the following command: ::
 
- export PLANNING_GROUP="panda_arm"
-
-Identify Link Numbers
-^^^^^^^^^^^^^^^^^^^^^
-You also need the link index numbers for the ``base_link`` and ``end_effector_link`` between which the IK will be calculated. You can count the number of links by viewing a list of links in your model: ::
+  rosrun moveit_kinematics auto_create_ikfast_moveit_plugin.sh --iktype Transform6D $MYROBOT_NAME.urdf <planning_group_name> <base_link> <eef_link>
```

Comment:
This currently fails on my machine with:

```
WARNING: The following packages cannot be authenticated!
  collada-dom2.4-dp-base collada-dom2.4-dp-dev collada-dom-dev
  ros-indigo-angles ros-indigo-urdf-parser-plugin ros-indigo-collada-parser
  ros-indigo-eigen-stl-containers ros-indigo-octomap ros-indigo-random-numbers
  ros-indigo-resource-retriever ros-indigo-geometric-shapes
  ros-indigo-tf2-msgs ros-indigo-tf2 ros-indigo-tf2-py ros-indigo-tf2-ros
  ros-indigo-tf ros-indigo-urdf ros-indigo-collada-urdf
E: There are problems and -y was used without --force-yes
```

Reply:
Also, this does not sufficiently cover the command line arguments for  `auto_create_ikfast_moveit_plugin.sh`

Reply:
I spent considerable time making sure that the reader could follow along with this tutorial line copy-pasting in commands for their own robot. I would like to maintain that usability.

Reply:
The failure is due to the ROS key change. I pushed a corresponding fix to master + melodic-devel as this also blocked Travis (https://github.com/ros-planning/moveit/pull/1503).

Reply:
This leaves out many of the command line arguments used by `auto_create_ikfast_moveit_plugin.sh` Please add a note that `--help` will provide documentation for the command line arguments.

Reply:
I opened an issue about the gpg key issue: https://github.com/personalrobotics/docker-public-images/issues/4

Reply:
I think we will have to wait until the docker is re-built before we can proceed with this PR

Reply:
No, not at all. We have a hot-fix for the GPG key issue via https://github.com/ros-planning/moveit/pull/1503.

Reply:
When I run this it fails. As a note $MYROBOT_NAME is set to panda_arm_hand as per the urdf I was using.
```
[ERROR] [1563206287.111715886]: Could not open file [robot.urdf] for parsing.
[ERROR] [1563206287.111754005]: failed to open urdf file robot.urdf
```
I also think that a quick explanation of where to use this command and possibly an example for the default panda arm config made 
```
rosrun moveit_kinematics auto_create_ikfast_moveit_plugin.sh --iktype Transform6D $MYROBOT_NAME.urdf panda_arm panda_link0 panda_link8
```